marklogic-data-hub-central/ui/e2e/cypress/e2e/mastering/stepsAndRunFlowValidations.cy.tsx

```diff
@@ -0,0 +1,227 @@
+import {Application} from "../../support/application.config";
+import loadPage from "../../support/pages/load";
+import runPage from "../../support/pages/run";
+import LoginPage from "../../support/pages/login";
+import {
+  entityTypeModal,
+  entityTypeTable,
+  propertyModal,
+  propertyTable,
+} from "../../support/components/model/index";
+import modelPage from "../../support/pages/model";
+import {confirmationModal, createEditStepDialog, toolbar} from "../../support/components/common/index";
+import {ConfirmationType} from "../../support/types/modeling-types";
+import curatePage from "../../support/pages/curate";
+import {
+  createEditMappingDialog,
+  mappingStepDetail
+} from "../../support/components/mapping/index";
+import browsePage from "../../support/pages/browse";
+
+const loadStepName = "loadAgent";
+const flowName1 = "testFlows1";
+const flowName2 = "testFlows2";
+const flowName3 = "testFlows3";
+const flowName4 = "testFlows4";
+const mapStep = "agentMap";
+const matchStep = "agentMatch";
+const mergeStep = "agentMerge";
+
+
+describe("Validate the scenarios when the steps are added in different flows", () => {
+
+  before(() => {
+    cy.visit("/");
+    cy.contains(Application.title);
+    cy.loginAsDeveloper().withRequest();
+    LoginPage.postLogin();
+  });
+  afterEach(() => {
+    cy.clearAllSessionStorage();
+    cy.clearAllLocalStorage();
+  });
+  after(() => {
+    cy.loginAsDeveloper().withRequest();
+    cy.deleteSteps("ingestion", "loadAgent");
+    cy.deleteSteps("mapping", "agentMap");
+    cy.deleteSteps("matching", "agentMatch");
+    cy.deleteSteps("merging", "agentMerge");
+    cy.deleteFlows(flowName1, flowName2, flowName3, flowName4);
+    cy.deleteEntities("Agent");
+    cy.deleteRecordsInFinal("loadAgent", "agentMap", "agentMatch", "agentMerge");
+    cy.deleteRecordsInFinal("sm-Agent-archived", "sm-Agent-mastered", "sm-Agent-merged", "sm-Agent-auditing", "sm-Agent-notification");
+    cy.resetTestUser();
+    cy.waitForAsyncRequest();
+  });
+  it("Create a load Step", () => {
+    toolbar.getLoadToolbarIcon().click({force: true});
+    cy.waitForAsyncRequest();
+    loadPage.stepName("ingestion-step").should("be.visible");
+    loadPage.loadView("th-large").click();
+    loadPage.addNewButton("card").click();
+    loadPage.stepNameInput().clear().type(loadStepName);
+    loadPage.saveButton().click();
+    cy.findByText(loadStepName).should("be.visible");
+  });
+  it("Add load Step to New Flow", {defaultCommandTimeout: 120000}, () => {
+    loadPage.addStepToNewFlow(loadStepName);
+    cy.waitForAsyncRequest();
+    cy.findByText("New Flow").should("be.visible");
+    runPage.setFlowName(flowName1);
+    loadPage.confirmationOptions("Save").click();
+    cy.waitForAsyncRequest();
+    //Fix for Windows, if accordion it's not collapsed
+    runPage.getPatientFlowAccordion(flowName1).then(($ele) => {
```

Comment:
"getPatientFlow..." is this an old bad copy/paste? Or we are patiently getting the accordion? :P
If possible I suggest we get this type of logic (opening if element is close) inside the "getFlowAccordion" to avoid having to duplicate it all the time.